contrib/ci/dependencies.xml

```diff
@@ -784,15 +784,15 @@
     </distro>
     <distro id="debian">
       <control>
-        <version>(>= 0.2.9)</version>
+        <version>(>= 0.3.3)</version>
```

Comment:
I think you also need to bump up the top level meson.build for this

Reply:
Hi @superm1,

I made this change, but this made the lgtm.com builder upset. Do you have some advice how to make that builder happy?

https://lgtm.com/projects/g/fwupd/fwupd/logs/rev/pr-567e623e129364154f1ea9ca31a3ac99d275c7ee/lang:cpp/stage:Build%20merge_a020dfb124810e01efe81b8a3e16c7aa4e684b00

Reply:
Probably modifications to https://github.com/fwupd/fwupd/blob/master/.lgtm.yml, but I don't know for sure.  Don't worry about this part for now, we can figure that out later when we're closer to merging.  I think it's more correct to have the top level `meson.build`.

Reply:
I think https://github.com/hughsie/libjcat/blob/master/.lgtm.yml is what you want to copy from.

Reply:
Hi @hughsie,

I wasn't able to figure out how to install this from pip, as libgusb isn't listed on https://pypi.org/. Any other suggestion?

`[2020-05-23 00:47:10] [build] |Subproject gusb finished.
[2020-05-23 00:47:10] [build] meson.build:198:0: ERROR: Version 0.3.2 of subproject dependency gusb already cached, requested incompatible version ['>= 0.3.3'] for dep gusb`

It's not clear to me why it goes to 0.3.2 of gusb, but not any further.

Reply:
Ah... I see what's happening!  it's because of `subprojects/libgusb.wrap`.  It's anchored at 0.3.2 (IE https://github.com/hughsie/libgusb/commit/43b327a1e213aff00833842c455a796a068077cd).  Update that to the commit matching 0.3.3 and the subproject fallback magic should start working and fix CI.

Reply:
@superm1, thanks! That did it! New builds will pass green now.